library/std/src/fs/tests.rs

```diff
@@ -906,7 +907,14 @@ fn read_link() {
         // junction
         assert_eq!(check!(fs::read_link(r"C:\Users\Default User")), Path::new(r"C:\Users\Default"));
         // junction with special permissions
-        assert_eq!(check!(fs::read_link(r"C:\Documents and Settings\")), Path::new(r"C:\Users"));
+        // Since not all localized windows versions contain the folder "Documents and settings" in english,
+        // we will briefly check, if it exists and otherwise skip the test. Except during CI we will always execute the test.
+        if Path::new(r"C:\Documents and settings\").exists() || env::var_os("CI").is_some() {
```

Comment:
It looks like this might've lowercased Settings by accident? It should probably be "Settings" still. (Also in the assert).

Reply:
Yes, this is correct. It was an accident and should be in uppercase as well.